hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/Trigger.java

```diff
@@ -80,9 +80,9 @@
     @JsonInclude(Include.NON_EMPTY)
     protected Map<String, String> tags;
 
-    /** A map with key based on actionPlugin and value a set of action's ids */
+    /** A list of links to actions represented by TriggerAction*/
     @JsonInclude(Include.NON_EMPTY)
-    private Map<String, Set<String>> actions;
+    private Set<TriggerAction> actions;
```

Comment:
I like this much better, using a Set of objects gives us a lot more flexibility.